### ISSUE TITLE
Bug 58122200: [React Native New Architecture]: Narrator and keyboard focus are not in sync with each other, when focus moves to edit field inside Clipboard.

### ISSUE THREAD

**Issue by vineethkuttan** [microsoft/react-native-gallery#619]
Build Details:
OS Version: Dev (OS Build 27881.1000)
App: React Native Gallery Experimental
AT: Screenreader

 

Repro-Steps:
Launch the React Native Gallery- Experimental app.
Navigate to the 'Expand menu' button and invoke it.
Navigate to 'Home' button and invoke it.
Navigate to 'Clipboard' and invoke it.
Navigate to the edit field.
Observe the screen reader announcement.
Actual Result:
When Narrator focus moves to any of the edit field in Clipboard, Narrator remains silent. Both Narrator and keyboard focus are not in sync with each other.

Expected Result:
Both Narrator and keyboard focus should be in sync with each other when focus moves to the edit field. And Narrator should not remains silent, it should announce about the focused element information.

User impact:

Users with visual impairment who rely on screen reader will face difficulties and might get confused if Narrator stays on silent.

MAS Reference: [MAS 4.3.1 – No Disruption of Accessibility Features](https://aka.ms/MAS4.3.1)
 
Trap ID: [1.4 - Uncomprehended Element](https://aka.ms/UncomprehendedElement)
 
“Have feedback on bugs logged, please tag bug as “A11yRCA” and add your feedback in the comment section.”

**Comment by vineethkuttan** [comment]
Issue is fixed
